### PR TITLE
Hails typo fixes

### PR DESCRIPTION
**Pull request by Corraban2**
## Summary
This pr fixes various typos I found in the hails. It also changes "favourite" to the American English "favorite".

### CODE REVIEW

File: data/human/hails.txt [hunk 1]
```diff
@@ -2538,7 +2538,7 @@ phrase "friendly pirate"
 		" You'll find some on the nearest pirate planet."
 		" You'll find some on the nearest anarchist planet."
 		" You'll find some on the nearest planet not oppressed by the man."
-		" You'll find some where ever the man does not go."
+		" You'll find somewhere ever the man does not go."
```
Comment (Ornok) [inline data/human/hails.txt:2541]: "some" actually refers to "contraband".

**Comment by movingpictures** [comment]
Why change the correct spelling of "favourite? that is used by more people throughout the world?

**Comment by MCOfficer** [comment]
> 
> 
> Why change the correct spelling of "favourite? that is used by more people throughout the world?

I can't find the reference for it, but ES uses US-American spelling.

File: data/human/hails.txt (hunk 1 above)
Comment (Corraban2) [reply data/human/hails.txt:2541]: Thanks, I didn't notice that earlier. Should it then become "You'll find some wherever the man does not go."?

Comment (Ornok) [reply data/human/hails.txt]: Yes, "wherever" is better. I wasn't sure about it, but in this case "where ever" doesn't fit.

**Pull request merged**